components/OAuth2.0/authorization.c

```diff
@@ -0,0 +1,283 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "authorization.h"
+#include "OauthHttpLocalServer.h"
+#include "SpiffsManger.h"
+#include "OauthAPICall.h"
+#include "cJSON.h"
+
+// ****************************** Global Variables //FIXME merge structs
+ServiceInterfaceHandler_t *pInterfaceHandler;
+ServiceInterfaceHandler_t ServiceInterfaceHandler;
```

Comment:
we have a service manager in another project with this name. please use the unique name related to your component!